src/background/background.js

```diff
@@ -91,6 +97,13 @@ chrome.runtime.onInstalled.addListener(function(details) {
   }
 });
 
+function selectTheme(info, tab) {
+  chrome.storage.local.set({
+    theme: info.menuItemId
+  });
+  chrome.tabs.sendMessage(tab.id, {action: "set_theme", content: info.menuItemId});
```

Comment:
Could instead use "refresh_theme" so that theme is only ever determined from local storage on receiving end

Reply:
Doesn't make much of a difference, let's stick to the current approach. :)